test-integration/externalUser.test.ts

```diff
@@ -214,15 +215,16 @@ describe('External User class', () => {
 
             expect(TONOMY_LOGIN_WEBSITE_requests.length).toBe(2);
             expect(payload.accountName).toBe(await (await TONOMY_ID_user.getAccountName()).toString());
+            expect(payload.username).toBe(await (await TONOMY_ID_user.getUsername()).username?.split('.')[0]);
```

Comment:
shouldnt this use `getBaseUsername()`?

Reply:
ohho let me remove this change from this change as i added this on username display task